cob_navigation_global/launch/2dnav_linear.xml

```diff
@@ -5,9 +5,9 @@
 
   <node pkg="cob_linear_nav" type="cob_linear_nav" respawn="false" name="move_base_linear" output="screen">
     <!-- remap inputs -->
-    <remap from="odom" to="/base_controller/odometry"/>
+    <remap from="odom" to="/base/odometry_controller/odometry"/>
     <!-- remap outputs -->
-    <remap from="cmd_vel" to="/base_controller/command_safe"/>
+    <remap from="cmd_vel" to="/base/twist_controller/command_safe"/>
```

Comment:
remove `/` prefix
(maybe also comments - for consistency with the other launch files)